main.go

```diff
@@ -153,9 +179,9 @@ func main() {
 	var seed []byte
 	if seedHex := os.Getenv(secretSeedEnvironmentVariable); seedHex != "" {
 		if logSecrets {
```

Comment:
Ugh, this variable should really be called "insecureLogSecrets" or something, but that's a complaint for another time.

Reply:
If you're not concerned about breaking backward compatibility, then the thing to do is to stop logging this value altogether, as I don't think there's any good reason to log the secret key to stdout.

Reply:
https://github.com/cloudflare/privacy-gateway-server-go/issues/65